### PR TITLE
Integration of search and API calls

### PR DESCRIPTION
**Pull request by kevinvongmany**

### CODE REVIEW

**Review by leontran44 — APPROVED** [review]
This is looking good but will we remove all chunks of code that got commented out?

File: Modal.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
             <div class="relative transform overflow-hidden rounded-lg bg-white text-left shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-lg">
                 <div class="bg-white px-4 pb-4 pt-5 sm:p-6 sm:pb-4">
                     <div class="sm:flex sm:items-start">
-                        <div class="mx-auto flex h-12 w-12 flex-shr class=" solid-green-100"ink-0 items-center justify-center rounded-full bg-green-100 sm:mx-0 sm:h-10 sm:w-10">
+                        <div class="mx-auto flex h-12 w-12 flex-shr" class=" solid-green-100"ink-0 items-center justify-center rounded-full bg-green-100 sm:mx-0 sm:h-10 sm:w-10">
```
Comment (Cbeattie97) [inline Modal.html:45]: Is this an accidental double quote?

**Review by Cbeattie97 — APPROVED** [review]
Wow, looks awesome! I've spotted what looks like an additional double quote. See the review comment. Happy to approve otherwise.

File: Modal.html (hunk 1 above)
Comment (kevinvongmany) [reply Modal.html:45]: This file's not in the final product, we copied it over to index.html! We did notice it and we fixed it up there so no worries.

**Pull request closed without merging**